External evaluation of Diffy Chat milestone 2

dastansam opened the pull request:
I wasn't able to reproduce actions from the tutorial video. More on this in the evaluation file.

Application: https://github.com/w3f/Grants-Program/blob/master/applications/Diffy_chat.md 
Milestone delivery PR: https://github.com/w3f/Grant-Milestone-Delivery/pull/912

keeganquigley commented:
I am experiencing the same lag issues. I also have trouble when I close the window and re-open it to register a new user. It seems to stick and not do anything.

In the chat app, I was never quite able to establish 2-way communication. It keeps asking to allow connection to <undefined>. Also, it doesn't seem to switch chat windows when clicking on the different users in the contact list.

Regarding entering the mnemonic phrase, as @dastansam mentioned, a user should never have to enter it into an app. I'm assuming that's just for test purposes, correct? Do you plan to integrate wallet functionality in the full product?

dastansam commented:
hey @keeganquigley,

@beoblock addressed the lag issues and I was able to confirm changes were made and they fix the issues from evaluation v1. I updated the evaluation file with new screenshots and changed the status to `Accepted`

keeganquigley commented:
Thanks @dastansam looks great as usual. I will forward your payment to the operations team.
